pkg/resourcemanager/vm/reconcile.go

```diff
@@ -119,23 +138,25 @@ func (g *AzureVirtualMachineClient) Ensure(ctx context.Context, obj runtime.Obje
 			switch azerr.Type {
 			case errhelp.AsyncOpIncompleteError:
 				instance.Status.Provisioning = true
+				instance.Status.Message = err.Error()
```

Comment:
This is already being set at the top of the error block.

Reply:
removed